test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -229,6 +232,33 @@ public void GetLastErrorNotIncludedInBulkGeneration()
         Assert.False(this.IsMethodGenerated("GetLastError"));
     }
 
+    /// <summary>
+    /// Verifies that the MINIDUMP_TYPE enum is generated.
+    /// </summary>
+    // TODO: Figure out why this enum is not being generated and get it to generate as it should be constant with every platform.
+    [Fact]
+    public void GetMINIDUMP_TYPEGeneration()
+    {
+        this.generator = new Generator(this.metadataStream, DefaultTestGeneratorOptions, this.compilation, this.parseOptions);
+        Assert.True(this.generator.TryGenerate("MINIDUMP_TYPE", CancellationToken.None));
+        this.CollectGeneratedCode(this.generator);
+        Assert.True(this.generator.TryGetEnumName("MINIDUMP_TYPE", out string? actualDeclaringEnum));
```

Comment:
You're using the wrong API here. `TryGetEnumName` can't be used to verify that the enum is generated. 
By adding `this.AssertNoDiagnostics();` to the test the generated code is logged as test output and I see that the enum is in fact generated. So no bug here, it seems.